mcp_pulseplate_server.py

```
@@ -0,0 +1,264 @@
+from typing import Dict, List, Any
+import asyncio
+import openai
+
+
+class PulsePlateMCPServer:
+    """MCP Server for PulsePlate project with ChatGPT integration"""
+
+    def __init__(self):
+        self.api_key = os.getenv("OPENAI_API_KEY")
+        if not self.api_key:
+            raise ValueError("OPENAI_API_KEY environment variable not set")
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)